hack/jenkins/benchmark-dockerized.sh

```diff
@@ -42,14 +42,9 @@ export KUBE_ROOT
 
 export PATH=${GOPATH}/bin:${PWD}/third_party/etcd:/usr/local/go/bin:${PATH}
 
-# Until all GOPATH references are removed from all build scripts as well,
-# explicitly disable module mode to avoid picking up user-set GO111MODULE preferences.
-# As individual scripts make use of go modules, they can explicitly set GO111MODULE=on
-export GO111MODULE=off
```

Comment:
does this assume hack/lib/golang.sh is used and GO111MODULE is unset, or do we need to unset to deal with any random env inherited from the user?

Reply:
Good question - MOST of our scripts call `kube::golang::setup_env` but clearly some do not (e.g. hack/make-rules/test-integration.sh).  We can "fix" those, but it also feels like we don't need to explicitly turn it OFF here.

If someone thinks they need this, and they set it, and it doesn't work at least we know that it's something they chose to try to do.

Since "on" is the default, I think we should assume the default.

Reply:
> Good question - MOST of our scripts call kube::golang::setup_env but clearly some do not (e.g. hack/make-rules/test-integration.sh). We can "fix" those, but it also feels like we don't need to explicitly turn it OFF here.

We should fix those. We try to be insulated from the host go env unless you explicitly ask to use it. (e.g. GOTOOLCHAIN=local)

Reply:
I will fix it in the case that brought this to light.